hedera-node/hapi-utils/src/main/java/com/hedera/node/app/hapi/utils/ethereum/EthTxData.java

```diff
@@ -477,6 +477,12 @@ private static EthTxData populateEip2390EthTxData(RLPItem rlpItem, byte[] rawTx)
                 );
     }
 
+    // Wrapping the bytes in a BigInteger to handle when there is a negative value. If we use the RLPItem.asBigInt()
+    // method it doesn't handle two's complement correctly.
+    private static BigInteger toValue(final byte[] value) {
+        return value.length == 0 ? BigInteger.ZERO : new BigInteger(value);
```

Comment:
Should we then use the `BigInteger` constructor with an explicit `signum` (see [here](https://github.com/openjdk/jdk/blob/master/src/java.base/share/classes/java/math/BigInteger.java#L481)):
```
return value.length == 0 ? BigInteger.ZERO : new BigInteger(1, value);
```